src/charm.py

```diff
@@ -75,7 +81,7 @@ class ReconcileRunnersEvent(EventBase):
     """Event representing a periodic check to ensure runners are ok."""
 
 
-CharmT = TypeVar("CharmT")
+CharmT = Any
```

Comment:
"Any" is a bit broad, is there another option here?

Reply:
Ah yes, thank you! Replaced with the actual charm type - by sarcrificing the generic function to be a little less generic :D